tools/docker/runner/start-agent

```diff
@@ -1,3 +1,3 @@
-${INSTALL_DIR}/bin/local_bus &
-${INSTALL_DIR}/bin/ieee1905_transport &
-${INSTALL_DIR}/bin/beerocks_agent
+#!/bin/sh
+${INSTALL_DIR}/scripts/prplmesh_utils.sh start -s -m a
+tail -f /dev/null # hack so the script will not exit forcing the container to stop
```

Comment:
It would also make sense to run the CLI as a foreground process instead. :ok:

Reply:
The agent doen't have a CLI